core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -110,7 +172,9 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging {
 
     def abortQueuedWorkflows(submissionId: UUID) = {
       datasource.inTransaction { dataAccess =>
-        dataAccess.workflowQuery.batchUpdateWorkflowsOfStatus(submissionId, WorkflowStatuses.Queued, WorkflowStatuses.Aborted)
+        workflowStatusCounter(WorkflowStatuses.Aborted).countDBResult {
```

Comment:
What do you think about updating counters at the `WorkflowComponent` level?  That way every call to `batchUpdateWorkflowsOfStatus()` would be instrumented without higher levels having to worry about it.